aQute.libg/src/aQute/lib/io/IO.java

```diff
@@ -984,6 +991,37 @@ public static void store(Object o, Writer w) throws IOException {
 		}
 	}
 
+	/**
+	 * Store output in a file but ensure that the content is updated atomically.
+	 * To ensure this, the file is first copied to a temporary file in the same
+	 * directory as the target. It is then renamed which will first attempt an
+	 * atomic move but will always replace.
+	 *
+	 * @param store the function provide the output
+	 * @param target the file to store it, parent directories will be created if
+	 *            necessary
+	 */
+	public static void store(ConsumerWithException<OutputStream> store, File target) throws Exception {
+
+		target.getParentFile()
+			.mkdirs();
+
+		File tmp = createTempFile(target.getParentFile(), target.getName(), ".tmp");
+		try {
+
+			try (FileOutputStream outputStream = new FileOutputStream(tmp)) {
```

Comment:
We stopped using `new FileOutputStream` due to some issues with it. Please use the `IO.outputStream(File)` method instead.`

Reply:
k